docs/index.pug

```diff
@@ -0,0 +1,11 @@
+extends /layout/base.pug
```

Comment:
I wonder whether it wouldn't be better if this was markdown. All the rest of the docs will be compiled from MD files, so it will look bad if we have one PUG file here. OTOH, PUG gives us more freedom for the index file, but I'd still go with MD. Any problems with that?